ocean/transects/compute_transects.py

```diff
@@ -31,41 +32,111 @@ def get_mask_short_names(mask):
   mask = mask.set_index(nTransects=['transectNames', 'shortNames'])
   return mask
 
-def compute_transport(output, timeavg, mesh, mask, name='Drake Passage'):
-  output = xr.open_mfdataset(output, concat_dim='Time', decode_times=False)
-  timeavg = xr.open_mfdataset(timeavg, concat_dim='Time', decode_times=False)
+def compute_transport(timeavg, mesh, mask, name='Drake Passage'):
   mesh = xr.open_dataset(mesh)
   mask = get_mask_short_names(xr.open_dataset(mask))
 
-  amask = mask.sel(shortNames=name).squeeze()
-  amask = amask.where(amask.transectEdgeGlobalIDs > 0, drop=True)
-  transectEdges = np.asarray(amask.transectEdgeGlobalIDs.values-1, dtype='i')
+  if name.lower() == 'all':
+    transectList = mask.shortNames[1:].values
```

Comment:
Should be a `:` instead of `1:` because the first name needs computed too, in my case with the 60to30 this was Drake Passage.

Reply:
fixed.